executors/node/plural_rules.js

```diff
@@ -0,0 +1,55 @@
+// Tests Intl Locale for pluralrules.
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale
+
+
+// TODO: Consider if a list of locales can be used.
+module.exports = {
+  testPluralRules: function (json) {
+    const label = json['label'];
+    const locale = json['locale'];
+
+    let return_json = {'label': label};
+
+
+    let test_options = {};
+    let sample;
+    if (json['sample']) {
+      sample = Number(json['sample']);  // Convert to a number
+    }
+    let plural_type;
+    if (json['type']) {
+      plural_type = json['plural_type'];
+      test_options['type'] = plural_type;
+    }
+
+    const supported_locales =
+          Intl.PluralRules.supportedLocalesOf(locale, test_options);
+    console.log("supported: ", supported_locales);
+    if (!supported_locales.includes(locale)) {
```

Comment:
If there is a way for `supported_locales` to be stored in a global scope, then you could also [convert it into a Set](https://stackoverflow.com/questions/28965112/javascript-array-to-set). I would expect a Set to have a faster lookup than an array, which would be O(n).

Reply:
Supported locales isn't a set of all hose supported by the API, but rather it's those that are compatible with the locale(s) entered. E.g., in the case of "iw", it returns "he", a more modern code for Hebrew. In the case of "mo", it gives "ro" (Romanian) instead of "Moldovan". Today's changes mark these locales as "unsupported".